receiver/envoyalsreceiver/internal/als/server.go

```diff
@@ -18,6 +18,8 @@ import (
 const (
 	apiVersionAttr = "api_version"
 	apiVersionVal  = "v3"
+	nodeAttr       = "node"
+	logNameAttr    = "log_name"
```

Comment:
Not immediately important since there are no semantic conventions around these, but we should consider making these attributes feel more OTel-native in the future.